### DISCUSSION TITLE
[v1.0.0b] - Tags, tools and categories sort alphabetically instead of by added time

### DISCUSSION THREAD

**Discussion by terrapinlogo** [mealie-recipes/mealie#1226] [Feature Request]
### First Check

- [X] This is not a feature request
- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the Mealie documentation, with the integrated search.
- [X] I already read the docs and didn't find an answer.

### What is the issue you are experiencing?

Tags, tools and categories sort by added time instead of alphabetically. The ability to chose how to sort would be nice.

### Deployment

Docker (Synology)

**Comment by hay-kot** [comment]
You mean on the recipe page?

**Reply by terrapinlogo** [reply]
On their respective pages. They are grouped together by first letter but otherwise sorted by added time
![Screenshot_20220519-063101~2](https://user-images.githubusercontent.com/5179945/169150883-01da32be-4310-4a1f-8c8f-107aa3574cc5.png)

**Comment by aberlaas** [comment]
Same here. tags, tools and categories are not alphabetically sorted so it's hard to quickly find an item.

**Comment by michael-genson** [comment]
I just [submitted a PR](https://github.com/hay-kot/mealie/pull/1354) to fix this.
